src/core/atomnames.h

```diff
@@ -61,9 +61,11 @@ item(_MARCO_TOGGLE_VERBOSE)
 item(_GTK_THEME_VARIANT)
 item(_GTK_FRAME_EXTENTS)
 item(_GTK_SHOW_WINDOW_MENU)
+item(_GTK_WORKAREAS)
 item(_MATE_PANEL_ACTION)
 item(_MATE_PANEL_ACTION_MAIN_MENU)
 item(_MATE_PANEL_ACTION_RUN_DIALOG)
+item(_MATE_WM_STRUT_AREA)
```

Comment:
To avoid renaming everything in MATE, or am i wrong?

Reply:
Btw, what is the adwantage to name it `_GNOME_WM_STRUT_AREA` ?

Reply:
What do you mean with everything? 

I am just speaking about new property - `_GNOME_WM_STRUT_AREA` / `_MATE_WM_STRUT_AREA`.

If you keep/use `_GNOME_WM_STRUT_AREA` you will be able to use gnome-panel with marco or will be able to use mate-panel with marco, metacity & compiz.

It will be ridiculous if both panels or wm will need support both property names that are used for one/same task.

Reply:
Alles klar :)
I will rename it.

Reply:
Just updated, better?